packages/edit-site/src/components/post-edit/index.js

```diff
@@ -139,17 +147,26 @@ function PostEditForm( { postType, postId } ) {
 }
 
 export function PostEdit( { postType, postId } ) {
+	const { ExperimentalBlockEditorProvider } = unlock(
+		blockEditorPrivateApis
+	);
+	const settings = usePatternSettings();
+	// Wrap everything in a block editor provider.
+	// This ensures 'styles' that are needed for the previews are synced
+	// from the site editor store to the block editor store.
 	return (
-		<Page
-			className={ clsx( 'edit-site-post-edit', {
-				'is-empty': ! postId,
-			} ) }
-			label={ __( 'Post Edit' ) }
-		>
-			{ postId && (
-				<PostEditForm postType={ postType } postId={ postId } />
-			) }
-			{ ! postId && <p>{ __( 'Select a page to edit' ) }</p> }
-		</Page>
+		<ExperimentalBlockEditorProvider settings={ settings }>
```

Comment:
Why not wrap the previews instead, for me this makes the dependency clearer.

Reply:
I think you misunderstood my suggestion :) 

I was asking to wrap the field "edit" component (or render) with `ExperimentalBlockEditorProvider` instead of wrapping the whole form. None of the other fields need this provider.

Reply:
Sorry! Gotcha!

I fixed with cbb8bda871392548d2595f40568cc95eaf6854e4. Another good way to fix it, it is updating the `usePostFields`: https://github.com/WordPress/gutenberg/blob/f891475bd3f69bcb66b1fdadff1e7173fdf5d424/packages/editor/src/components/post-fields/index.ts#L26-L73.


The issue is that `usePatternSettings` isn't defined in the `@wordpress/editor` package. I'm happy to proceed with a refactor if you think that it makes sense.